ls1intum/Artemis
# `Development`: Rename monitoring to live statistics

### Checklist
#### General



### Motivation and Context



As discussed with @bassner, we need to limit the subscription of the exam live statistics. Therefore i adapted the endpoints in order to match the other exam endpoints and to limit the subscription only to instructors.

Additionally, we renamed monitoring to live statistics.

### Description


- Renamed the endpoints from `/topic/exam-monitoring/..` to `/topic/exams/..`.
- Renamed everything related to monitoring to live statistics


### Steps for Testing

Prerequisites:
- 1 Instructor
- One/Multiple Students
- Exam

1. Log in to Artemis
2. Create an Exam
3. Register multiple students
4. Navigate to the Exam Live Statistics dashboard
5. Enable the Exam Live Statistics
6. Participate with the students in the exam
7. Verify that actions are collected
8. Verify that everything works as before

### Review Progress





#### Code Review
#### Manual Tests
